### DISCUSSION TITLE
Different test result between locally build and command-installed pyright

### DISCUSSION THREAD

**Discussion by Andres981231** [microsoft/pyright#3054] [Q&A] [answered]
I have built pyright locally with package.json in pyright-internal directory, then I run the locally build pyright with comman **node pyright.js \<file\>**, and the result I got produces lots of unreasonable errors like **error: "str" is not defined (reportUndefinedVariable)**. However in the command-installed(by pip or npm) pyright, these errors will not occur. So I wonder whether there is any additional configuration needs to be done when building pyright locally in order to lead to the same correct running result.

**Comment by erictraut** [comment, marked as answer]
Please follow the instructions here: https://github.com/microsoft/pyright/blob/main/docs/build-debug.md.

**Reply by Andres981231** [reply]
Thanks! My problem has been solved.
